contributing/index.md

```diff
@@ -1,45 +1,74 @@
 (sprint_help)=
-# Contribute
+# Contribute to the events
 
-% TODO: make nicer and add missing pages/sections
+:::{toctree}
+:maxdepth: 1
+:hidden:
 
-## ...with the website
-To build the website locally, install the requirements
+self
+../about/contributing_to_pymc/index
+../about/contributing_to_documentation/docs_presentation
```

Comment:
I would link to the index page instead so readers can choose to read, watch the video or both.

Reply:
Done!